### ISSUE TITLE
Fix integration test "FreeText Editor FreeText rotation must check that the dimensions of a rotated annotations are correct after a font size change"

### ISSUE THREAD

**Issue by timvandermeij** [mozilla/pdf.js#16968]
Introduction date: July 4, 2023
Introduction PR: https://github.com/mozilla/pdf.js/commit/1ce6668a7071178bd08e5b817473eea780cd4d07 (/cc @calixteman)
Type: intermittent (sometimes it works, indicating a possible timing/waiting issue)
OS: Windows
Logs: http://54.193.163.58:8877/d67e98b920493b4/output.txt
Traceback:

```
Failures:
1) FreeText Editor FreeText rotation must check that the dimensions of a rotated annotations are correct after a font size change
  Message:
[31m    In firefox: Expected false to equal true.[0m
  Stack:
        at <Jasmine>
        at C:\pdfjs\botio-files-pdfjs\private\d67e98b920493b4\test\integration\freetext_editor_spec.js:1341:14
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async Promise.all (index 0)
        at async UserContext.<anonymous> (C:\pdfjs\botio-files-pdfjs\private\d67e98b920493b4\test\integration\freetext_editor_spec.js:1273:7)
```

**Comment by Snuffleupagus** [comment]
@calixteman Was this completely fixed by PR #17036, or is there more work to do here?

**Comment by calixteman** [comment]
Normally the mentioned PR fixed it.
